sympy/printing/codeprinter.py

```diff
@@ -200,7 +200,7 @@ def _doprint_loops(self, expr, assign_to=None):
         if text != lhs_printed:
             lines.extend(openloop)
             if assign_to is not None:
-                text = self._get_statement("%s = %s" % (lhs_printed, text))
+                text = self._get_statement("{} = {}".format(lhs_printed, text))
```

Comment:
When `pyupgrade` was previously used it was decided not to change `%s` to format (there is an option to disable that aspect).

I don't necessarily mind changing it now but would it make sense to go to f-strings rather than format?
```python
                # Which of these?
                text = self._get_statement("%s = %s" % (lhs_printed, text))
                text = self._get_statement("{} = {}".format(lhs_printed, text))
                text = self._get_statement(f"{lhs_printed} = {text}")
```

Reply:
I agree f-strings would be even better in many cases, although `format` or `%` may be preferred if the formatting arguments are large and would make the f-string hard to read. 

I don't have a preference between `format` vs. `%`. I think pyflakes can catch errors with both, so we don't really need to worry about that.

Reply:
I generally prefer f-string, unless the string contains `{}`, eg latex. In that case format function is even worse than f-string. `%s` seems the only readable choice.

Reply:
I agree let's continue to use `%` for LaTeX and anywhere else `{}` is already part of the string (maybe in some of the set printers?).